CLAUDE.md

```
@@ -90,9 +93,10 @@ async with ctx.scope(
 ):
 ```
 
-**Protocol-Based Extensions:**
+**Protocol-Based Extensions (use state methods):**
 ```python
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)